test/runtests.jl

```diff
@@ -1,7 +1,19 @@
 using Compat, StatsBase
 using Compat.LinearAlgebra
-using Compat.Random
 
+@static if StatsBase.BASESTATS_IN_STATSBASE
+    using Random
+    const Compatvar = var
```

Comment:
Since `var` and `Compat.var` are different functions on 0.6 I couldn't come up with a better solution. How did you mean?
I don't think it matters much, presumably `StatsBase` will drop 0.6 when 0.7 is released anyway, and then these will all just be `var` without `Compat` being involved.

Reply:
I was thinking of something like `const var = StatsBase.var` in one branch, and `const var = Compat.var` in the other one. The point is precisely to limit as much as possible the changes that will have to be applied manually when dropping 0.6 support.

Reply:
But 
```
const var = Compat.var
```
would mean that `StatsBase` extend the unexported `Compat.var` and people will have to use a fully qualified name for this. In addition, that would mean `StatsBase` would export the unexported `Compat.var` and break things...

Reply:
This is just a test file, it doesn't affect what users see at all AFAICT.

Reply:
Ah, right, was thinking about the same case we have in the source file. But again, I don't think we can do that since they are two different functions, and we use them both.

Reply:
OK, let's keep it that way then.